distributed/tests/test_worker.py

```diff
@@ -1497,6 +1497,22 @@ def test_resource_limit(monkeypatch):
                 assert all("127.0.0.1" == d["host"] for d in info["workers"].values())
 
 
+@pytest.mark.asyncio
+@pytest.mark.parametrize("Worker", [Worker, Nanny])
+@pytest.mark.parametrize("protocol", ["tcp", "ucx"])
```

Comment:
```suggestion
@pytest.mark.parametrize("protocol", ["ucx"])
```

I recommend that we remove tcp here to keep tests somewhat fast.